master/static/migrations/20210301160616_add_trial_profiler_metrics.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE trial_profiler_metrics (
+    values  float4[]    NOT NULL,
```

Comment:
I'm curious why it's better to store the metrics in postgres as batches rather than a row per individual metric. Is there a performance implication?

Reply:
Yeah, well if you go a row per metric then you're also storing all the labels per metric too. That was the main concern, but yeah, I expect 10 rows of batches of 100 to do better than 10k rows also (the index scan has to gather a lot less).